modules/ROOT/pages/_partials/commons/common-document.adoc

```diff
@@ -263,7 +263,7 @@ ifdef::is-android[]
 [NOTE]
 --
 Care should be taken when storing and recovering data in a document or converting that document to JSON and back. +
-Data encoding (Fleece) can result in `Long` values being converted to `Float` instead of `Double`. +
```

Comment:
I am so sorry: I just noticed that this is backwards.  Needs correction here and in the backport.  It should read:

Data encoding (Fleece) can result in `Long` values being converted to `Integer`, and `Double` values to `Float`.

Reply:
Done!